web/src/beta/components/Resizable/hooks.ts

```diff
@@ -99,12 +96,14 @@ export default (
     window.removeEventListener("touchend", onResizeEnd);
   }, [onResize, onResizeEnd]);
 
-  unbind = unbindEventListeners;
-
   useEffect(() => {
+    if (!isResizing) {
+      unbindEventListeners();
+      return;
+    }
     bindEventListeners();
     return () => unbindEventListeners();
-  });
+  }, [bindEventListeners, isResizing, unbindEventListeners]);
```

Comment:
Aren't we going to get MANY bindEventListeners calls with these deps added?

My understanding is:

As the panel gets larger or smaller, the `onResize` useCallback will be updated because it has `size` as a dep. Because `onResize` gets updated, `bindEventListeners` and `unbeindEventListeners` will be updated, which in the end will make this useEffect run again, running `bindEventListeners` again which will add more event listeners to the window.

If you think that logic is incorrect, please let me know. But that is my understanding, so I think leaving this useEffect as it was with no deps is correct.

Reply:
Added PR description.
I understand what you want to say, but working with both before/after is almost the same.
And adjusting `useEffect` is not main purpose of this PR.